Sources/BraintreeShopperInsights/BTShopperInsightsClientV2.swift

```diff
@@ -33,6 +33,15 @@ public class BTShopperInsightsClientV2 {
     
     // MARK: - Public Methods
     
+    /// This method creates a new customer session.
+    /// - Parameters:
+    ///    - request: A `BTCustomerSessionRequest`
+    /// - Returns: A `String` representing a session ID if successful
+    /// - Throws: An error if the request fails for some reason or if the response is invalid.
+    public func createCustomerSession(request: BTCustomerSessionRequest) async throws -> String {
+        try await BTCreateCustomerSessionAPI(apiClient: apiClient).execute(request)
```

Comment:
Take it or leave it:
```suggestion
        let createCustomerSessionAPI = BTCreateCustomerSessionAPI(apiClient: apiClient)
        try await createCustomerSessionAPI.execute(request)
```